packages/cf-component-page/src/PageHeader.js

```diff
@@ -6,7 +6,7 @@ class PageHeader extends React.Component {
       <header className="cf-page__header">
         <h1 className="cf-page__title">{this.props.title}</h1>
         {this.props.subtitle &&
-          <p className="cf-page__subtitle">{this.props.subtitle}</p>}
+          <h2 className="cf-page__subtitle">{this.props.subtitle}</h2>}
```

Comment:
I like the idea of using `cf-component-heading`, but it would introduce a dependency in `cf-component-page`